driver_test.go

```diff
@@ -274,3 +275,31 @@ func TestReplaceCanOverwrite(t *testing.T) {
 	require.NoError(t, row.Scan(Document(&movie)))
 	require.Equal(t, updatedPrisioners, movie)
 }
+
+func TestCreateTable(t *testing.T) {
+	sess := fixtures.SetUp(t)
+	client := dynamodb.New(sess)
+	_, _ = client.DeleteTable(&dynamodb.DeleteTableInput{TableName: aws.String("movies")})
+	defer func() {
+		_, _ = client.DeleteTable(&dynamodb.DeleteTableInput{TableName: aws.String("movies")})
+	}()
+	driver, err := New(Config{Session: sess}).OpenConnector("")
+	require.NoError(t, err)
+	db := sql.OpenDB(driver)
+	err = db.Ping()
+	require.NoError(t, err)
+	_, err = db.Exec(`
+		CREATE TABLE movies (
+			title STRING HASH KEY,
+			year NUMBER RANGE KEY,
+			director STRING,
+			
+			PROVISIONED THROUGHPUT READ 1 WRITE 1,
```

Comment:
I think this should be outside the `()` since it's at the table level and not directly related to columns, similar to how PARTITION BY clauses for tables in SQL are outside the `()`

Reply:
I was wondering about that!